nncf/torch/tensor.py

```diff
@@ -30,14 +30,24 @@ def concatenate(cls, tensors: List[NNCFTensor], axis: int) -> NNCFTensor:
         return PTNNCFTensor(ret_tensor)
 
     @classmethod
-    def ones(cls, shape: List[int], device) -> NNCFTensor:
+    def ones(cls, shape: Union[int, List[int]], device: torch.device) -> NNCFTensor:
         return PTNNCFTensor(torch.ones(shape, device=device))
 
     @classmethod
-    def check_all_close(cls, tensors: List[NNCFTensor]) -> None:
+    def allclose(cls, tensors: List[NNCFTensor]) -> None:
```

Comment:
I'm being picky here, but torch.allclose and numpy.allclose are not expected to assert, they just return bool values.
It would be more clear if assert happens outside of this method, or method is renamed to `assert_allclose`, follow the example of `np.testing.assert_allclose`

Reply:
Agree, done